test/integration/minimal/controls/minimal.rb

```diff
@@ -74,7 +74,7 @@
     end
 
     it 'has a timeout length of 1 second' do
```

Comment:
Looks like this is the default 
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_region_backend_service#timeout_sec
and we dont set it per https://github.com/terraform-google-modules/terraform-google-lb-internal/blob/6934d4e528c1ec808e258bd4455b1fcc5d71d642/main.tf#L51
```suggestion
    it 'has a timeout length of 30 second' do
```